fees/arweave/index.ts

```diff
@@ -0,0 +1,47 @@
+import type { FetchOptions, } from "../../adapters/types";
+import { Adapter, ProtocolType } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+const axios = require('axios');
+
+
+const fetch = async (_a: any, _b: any, options: FetchOptions) => {
+    const dayTimestamp = options.startOfDay * 1000;
+
+    const { data } = await axios.get('https://api.viewblock.io/arweave/stats/advanced/charts/txFees?network=mainnet', {
+        headers: {
+            'origin': 'https://arscan.io'
+        }
+    });
+
+    const timestamps = data.day.data[0];
+    const fees = data.day.data[1];
+
+    const index = timestamps.findIndex((ts: number) => ts === dayTimestamp);
+    const tokenAmount = index !== -1 ? parseFloat(fees[index]) : 0;
```

Comment:
would prefer adapter failing instead of returning incorrect result if record is not found